source/testing-your-html/index.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: Test your HTML matches GOV.UK Frontend
+weight: 61
+---
+
+# Test your HTML matches GOV.UK Frontend
+
+You can use our test fixtures to check you're outputting the same HTML that GOV.UK Frontend uses, when you:
+
+- create your own component macros (also called ‘partials’ or ‘templates’)
+- update GOV.UK Frontend
+
+## Using the HTML test files
+
+If you [installed GOV.UK Frontend with Node.js package manager (npm)](/installing-with-npm), you can find the `fixtures.json` file for each component in the `node_modules/govuk-frontend/govuk/components/COMPONENT-NAME/` folder, where `COMPONENT-NAME` is the name of the component.
+
+<%= warning_text('You should not test your HTML using the YAML files in the <code>src/</code> folder.') %>
+
+For example, you can find the `fixtures.json` file for the button component in the `node_modules/govuk-frontend/govuk/components/button/` folder:
+
+```js
+[
```

Comment:
> Looks good 👍 Just left one comment.
> 
> We could also consider adding something the package README to document the HTML fixtures since it already [explains](https://github.com/alphagov/govuk-frontend/blob/master/package/govuk/components/accordion/README.md#component-options) what the macro options file is for. I can create another issue for that if it's something we want to do?

Hi @hannalaakso - ah thanks for spotting this. Are those individual READMEs in each component folder auto-generated?

Reply:
Thanks Vanita - fixed in ee2717b.